test-util/src/main/java/io/camunda/zeebe/test/util/bpmn/random/blocks/AbstractBlockBuilder.java

```diff
@@ -26,6 +30,12 @@ protected AbstractBlockBuilder(final String prefix, final String elementId) {
     this.prefix = prefix;
   }
 
+  @Override
+  public AbstractFlowNodeBuilder<?, ?> buildFlowNodes(
+      final AbstractFlowNodeBuilder<?, ?> nodeBuilder) {
+    return nodeBuilder.manualTask(elementId).name(elementId);
```

Comment:
❌ Let's keep this in `ManualTaskBlockBuilder` class and create a new method in `AbstractFlowNodeBuilder`. Afterwards, we can call it here as `nodeBuilder.abstractTask(elementId).name(elementId);`. This way, the implementation will be aligned with other elements as well.